server/src/routes/api/v1/usersRouter.js

```diff
@@ -17,4 +19,32 @@ usersRouter.post("/", async (req, res) => {
   }
 });
 
+usersRouter.get("/image", async (req, res) => {
+  try {
+    const userToReturn = await User.query().findById(req.user.id)
+    return res.status(200).json({ photo : userToReturn.imageUrl })
+  } catch (error) {
+    return res.status(500).json({ errors: error })
+  }
+})
```

Comment:
this route already exists in the user-sessions router! delete this route and just use `/api/v1/user-sessions/current`